pylib/Utilities/Copytree.py

```diff
@@ -52,7 +52,7 @@ def execute(self, log, keyvals, testDef):
             log['stderr'] = "Src directory not specified"
             return
         # define the dst directory
-        dst = os.path.join(testDef.options['scratchdir'], log['section'])
+        dst = os.path.join(testDef.options['scratchdir'], log['section'].replace(":","_"))
```

Comment:
Oh sorry -- one question: does `.replace(a,b)` replace *all* instances of `a` with `b`?  Or only the first instance?  (sorry, I don't know Python enough to know which way this particular method uses)